static/sass/patterns/_gallery.scss

```diff
@@ -88,3 +88,30 @@ $min-image: 300px;
   grid-column: 1 / -1;
   margin: size('shim') 0;
 }
+
+
+// Videos
+// ------
+
+.video-gallery {
+  display: flex;
+  justify-content: space-evenly;
```

Comment:
I don't know if it matters, but MDN says this is supports only in Chrome 60+ and Firefox 52.